tests/mapping/constant_keyword.yaml

```diff
@@ -0,0 +1,33 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test constant_keyword field type and retrieving a doc.
+
+prologues:
+  - path: /{index}
+    method: PUT
+    parameters:
```

Comment:
Btw, in prologues you can do `path: /movies` and not use `parameters`. That's because we don't test schema here. Makes it a little shorter, NBD.